easybuild/easyconfigs/i/IPM/IPM-2.0.6-CrayGNU-2016.11.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'ConfigureMake'
+
+name = 'IPM'
+version = '2.0.6'
+# versionsuffix = '-debug'
+# cudaversion =  '8.0'
+# versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'https://github.com/nerscadmin/IPM'
+description = """Integrated Performance Monitoring for High Performance
+Computing"""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'verbose': False}
+# https://github.com/hpcugent/easybuild-easyconfigs/pull/2928#issuecomment-216220341
+
+source_urls = ['https://github.com/nerscadmin/IPM/archive']
+sources = ['%(version)s.tar.gz']
+checksums = [ 'c01207ca942bef74c7d48df965fca99f' ]
+patches = [ 'IPM-2.0.6-CrayGNU-2016.11.patch' ]
+
+# ploticus+mxml required for parser
+# ploticus is in /apps/common/system/
+dependencies = [
+    ('ploticus', '2.42-ipm', '', True),
+    ('mxml', '2.10', '', True),
+    ('libunwind', '1.2', '', True),
+    ('papi/5.5.0.2', EXTERNAL_MODULE),
+]
+
+preconfigopts  = ' ./bootstrap.sh && '
+preconfigopts += ' sed -i -e \'s/-qversion/-qversion --version/g\' configure && '
+buildopts = ' MPICC=cc MPIF77=ftn MPIFC=ftn FC=ftn '
+configopts  = ' --with-papi=/opt/cray/pe/papi/5.5.0.2'
```

Comment:
Changes in the metadata files are always reviewed and therefore the probability of someone breaking your build is pretty low. FYI most builds that rely on Cray modules currently depend on the external metadata file.

BTW this is not the appropriate place for asking questions about existing entires on metadata file (neither for questioning its existence), please read the [cug paper](https://cug.org/proceedings/cug2016_proceedings/includes/files/pap145.pdf) for more details.

1. `FFTW_DIR = /opt/cray/pe/fftw/3.3.4.11/haswell/lib` 
   * (and we need the parent folder for the includes)
2. `SLURMDIR` is not for Cray (we use it for leone & mönch)
3. `PATH` or `MANPATH` should work